plugins/package-curation-providers/clearly-defined/build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2023 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+plugins {
+    // Apply core plugins.
+    `java-library`
+}
+
+dependencies {
+    api(project(":plugins:package-curation-providers:api"))
```

Comment:
Isn't the API that this plugin implements just an `implementation` dependency? Similar for the other plugins.

Reply:
The factory implements inherit from the `PackageCurationProviderFactory` from the `api` module, shouldn't it be an api dependency then?

Reply:
Looks like you are right, as the [docs](https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_recognizing_dependencies) mention "types used in super classes or interfaces".